src/auro-input.js

```diff
@@ -127,20 +251,23 @@ export default class AuroInput extends BaseInput {
         id="${this.id}"
         name="${ifDefined(this.name)}"
         type="${this.getInputType(this.type)}"
+        inputmode="${ifDefined(this.numericKeyboard ? `numeric` : undefined)}"
+        pattern="${ifDefined(this.numericKeyboard ? `[0-9]` : undefined)}"
```

Comment:
I don't think we need ifDefined here, since undefined is already the else case in the ternary

Reply:
When ifDefined is removed than all inputs that don't use the numeric keyboard have `inputmode="undefined" pattern="undefined"`. When using ifDefined the attributes are not added at all.